modules/em/test/test_bayesem3d_ccc.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import print_function
+import IMP
+import IMP.test
+import IMP.em
+import os, sys
+import numpy as np
+import random, math
+
+def get_particles(m, input_pdb):
+    ### Read input fragment
+    sel0 = IMP.atom.BackbonePDBSelector()
+    sel1 = IMP.atom.CBetaPDBSelector()
+    mh = IMP.atom.read_pdb(input_pdb, m, IMP.atom.OrPDBSelector(sel0, sel1))
+    IMP.atom.add_radii(mh)
+    return mh, IMP.core.get_leaves(mh)
+
+class Tests(IMP.test.TestCase):
+    """Derivative score stability Test"""
+
+    def setUp(self):
+
+        IMP.test.TestCase.setUp(self)
+
+
+        IMP.set_log_level(IMP.SILENT)
+        IMP.set_check_level(IMP.NONE)
+
+
+        self.m = IMP.Model()
+        self.mh, self.ps = get_particles(self.m, "input/input_atom.pdb")
+        IMP.atom.create_rigid_body(self.mh)
+        self.prot_rb = IMP.core.RigidMember(IMP.core.get_leaves(self.mh)[0]).get_rigid_body()
+
+        ##Read and setup EM
+        self.res = 5.
+        voxel_size = 1.5
+
+        dmap = IMP.em.read_map("input/sampled_all.mrc", IMP.em.MRCReaderWriter())
```

Comment:
Use `self.get_input_file_name` here, not `input/` directly.  You can't guarantee that the cwd is the directory containing the test (usually it isn't.)